src-js/optimizelyClient.js

```diff
@@ -0,0 +1,49 @@
+import * as optimizelySDK from '@optimizely/optimizely-sdk';
+const COOKIE_KEY = 'cci-org-analytics-id';
+
+class OptimizelyClient {
+  constructor() {
+    this.client = optimizelySDK.createInstance({
+      datafile: window.optimizelyDatafile,
+    });
+  }
```

Comment:
so no longer doing the localstorage thing?

Reply:
Nope! no need 😊